packages/opentelemetry-remote-sampler-aws-xray/src/remote-sampler.types.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+// cast to interface type to validate JSON 
+// e.g: const jsonData = {}
+// const rule: SamplingRule = JSON.parse(jsonData) as SamplingRule;
+export interface SamplingRule {
```

Comment:
If those types are following some aws official documentation for how the response is structured, it could be assume to add a link here to help with future maintenance and readability.